src/cmd/create.go

```diff
@@ -710,16 +710,33 @@ func pullImage(image, release, authFile string) (bool, error) {
 		shouldPullImage = true
 	}
 
+	imageSizeChan := make(chan string)
+
 	if promptForDownload {
+
 		fmt.Println("Image required to create toolbox container.")
+		// Launch a goroutine to call getImageSizeFromRegistry and send the size to the channel
+		go func() {
+			imageSize, err := getImageSizeFromRegistry(imageFull)
+			if err != nil {
+				logrus.Debugf("Getting image size failed: %s", err)
+				imageSize = ""
+			}
+			imageSizeChan <- imageSize
+		}()
 
-		var prompt string
+		// Show the download question immediately
+		prompt := fmt.Sprintf("Download %s (...)? [y/N]:", imageFull)
+		fmt.Println(prompt)
 
-		if imageSize, err := getImageSizeFromRegistry(imageFull); err != nil {
-			logrus.Debugf("Getting image size failed: %s", err)
-			prompt = fmt.Sprintf("Download %s? [y/N]:", imageFull)
-		} else {
+		// Wait until the size is received
+		imageSize := <-imageSizeChan
```

Comment:
This means that the user can't respond to the prompt before the `skopeo inspect` returns, which is odd.  The user should be able to respond `yes` or `no` regardless of whether `skopeo inspect` has returned or not.

Reply:
This is resolved because the user can respond to the prompt before the `skopeo inspect` returns.